tools/Resources.py

```diff
@@ -0,0 +1,60 @@
+import os
+import requests
+from urllib3 import disable_warnings, exceptions
+from requests.auth import HTTPBasicAuth
+
+
+class Resources:
+
+    def get_resources(self, target, **kwargs):
+        url = "https://" + target + "/suite-api/api/resources"
+        querystring = {
+            'parentId': kwargs['parentid'],
```

Comment:
looks like you are relying on parentid and resourcekind here, provided as kwargs. You should rather explicitly expect them, otherwise this run will fail, won't it?

Reply:
resolved